bucket_map/src/index_entry.rs

```diff
@@ -83,21 +83,38 @@ impl BucketOccupied for BucketWithHeader {
 #[derive(Debug)]
 pub struct IndexBucketUsingBitVecBits<T: PartialEq + 'static> {
     /// 2 bits per entry that represent a 4 state enum tag
-    pub enum_tag: BitVec,
+    pub enum_tag_first_bit: BitVec,
+    /// second will be empty in all healthy cases because in real use, we only use enum values 0 and 1
```

Comment:
Should the comments be "we only use emum values 0 and 2", i.e. Free or OneSlot?

Reply:
yes, this comment is wrong. I fixed it.